itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationOptimisedBootstrap.java

```diff
@@ -125,14 +130,8 @@ public void testBuildTableDiffGeneration() throws Throwable {
         .run("repl status " + replicatedDbName)
         .verifyResult(tuple.lastReplicationId)
         .run("use " + replicatedDbName)
-        .run("show tables like 't1'")
-        .verifyResult("t1")
-        .run("show tables like 't2'")
-        .verifyResult("t2")
-        .run("show tables like 't1_managed'")
-        .verifyResult("t1_managed")
-        .run("show tables like 't2_managed'")
-        .verifyResult("t2_managed")
+        .run("show tables")
```

Comment:
is this change needed?

Reply:
We were making unnecessary read calls, even though only one read is required for verification.